### ISSUE TITLE
Whether the training code can be provided?

### ISSUE THREAD

**Issue by dearhoper** [adithyaselv/face-expression-detect#1]
Hi, 
When I tried to use your code to test our own emotional pictures, most of them were not correctly identified. Even if the emotion of the picture is obvious.

![Image](https://thumbnail0.baidupcs.com/thumbnail/14f16bd824fa931e631c27755fd55b3a?fid=202133577-250528-575618038128054&time=1488848400&rt=sh&sign=FDTAER-DCb740ccc5511e5e8fedcff06b081203-Prn%2FIABSPg2uXfKSEyV3WjkuNZw%3D&expires=8h&chkv=0&chkbd=0&chkpc=&dp-logid=1517057643368696747&dp-callid=0&size=c710_u400&quality=100)

I would like to try to train our own facial expression database. Can you provide the training code? Thanks.

**Comment by kaashmonee** [comment]
Yeah, I would love to know how you trained it. Did you write your own neural network?
